plugin.audio.rautemusik/addon.xml

```diff
@@ -1,25 +1,33 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.audio.rautemusik"
-	name="RauteMusik"
-	version="0.3.0"
-	provider-name="Leo Moll">
+	name="RauteMusik.FM"
+	version="0.5.0"
+	provider-name="RauteMusik GmbH, Leo Moll">
 	<requires>
-		<import addon="xbmc.python" version="2.1.0"/>
+		<import addon="xbmc.python" version="2.14.0"/>
+		<import addon="xbmc.addon" version="13.0.0"/>
```

Comment:
Please drop xbmc.addon and xbmc.json. You don't need to set them

Reply:
Fixed!